src/hooks/recoveryMethods.js

```diff
@@ -13,8 +13,8 @@ export function useRecoveryMethods(accountId) {
     const dispatch = useDispatch()
 
     useEffect(() => {
-        if (accountId === wallet.accountId) {
-            dispatch(loadRecoveryMethods(accountId))
+        if (accountId === wallet.getAccountId()) {
```

Comment:
any reason not to use `wallet.accountId`? I think `.getAccountId()` was removed.